UPGRADE.md

```diff
@@ -45,6 +45,10 @@ New rule:
 The documentation did not mention the `SourceIP` type, but it was possible to create a policy using it instead of `Connection`. Since `SourceIP` 
 is not a correct value, we have decided to deprecate it. If you are using `SourceIP` in your policy, please update it to use `Connection` instead.
 
+#### Changes on revoking dataplane tokens
+
+Revoking a dataplane token on Unversal mesh clusters now requires restarting the mesh control plane, otherwise connected dataplanes will still be able to communicate until they restart/reconnect to the control plane.  
```

Comment:
```suggestion
Revoking a data-plane token on Universal zones now requires restarting the mesh control plane, otherwise connected data-planes will still be able to communicate until they restart/reconnect to the control plane.
```

I'm not sure this is correct. I think the change is more:

> CP/DP authentication is only checked at connection start. This means that if a token expires or is revoked after the data-plane connects, the connection won't stop. The recommended action on token revocation is to either restart the control-plane or the concerned data-planes

Reply:
Improved. I see more places using `dataplane` than `data-plane`, so I'm using the more common one.